migrations/versions/rev-2021-01-09-18:31:24-cd8488c19b16_.py

```diff
@@ -0,0 +1,31 @@
+"""empty message
+
+Revision ID: cd8488c19b16
+Revises: fb8dd6aed9a9
+Create Date: 2021-01-09 18:31:24.557692
+
+"""
+
+from alembic import op
+import sqlalchemy as sa
+import sqlalchemy_utils
+
+
+# revision identifiers, used by Alembic.
+revision = 'cd8488c19b16'
+down_revision = 'fb8dd6aed9a9'
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('events', sa.Column('is_demoted', sa.Boolean(), nullable=False, server_default=False))
```

Comment:
```suggestion
    op.add_column('events', sa.Column('is_demoted', sa.Boolean(), nullable=False, server_default='False'))
```

server_default needs to be quoted